client/src/app/site/motions/components/motion-detail/motion-detail.component.ts

```diff
@@ -1170,6 +1218,87 @@ export class MotionDetailComponent extends BaseViewComponent implements OnInit {
         this.repo.followRecommendation(this.motion);
     }
 
+    /**
```

Comment:
Please make this more generic and move this into a service! This service just need a small interface: startEditing(<identifier, that we talk about motions>): <some kind of subscription, so the service can keep track of this> and stopEdition(<identifier>, <subscription?>, <reason>) which cleans up all subscriptions and sends the final notify message.
We maybe want to use this in other edit forms, too. And this component is already crowded enough..